tests/dhcp_relay/test_dhcp_pkt_fwd.py

```diff
@@ -0,0 +1,329 @@
+import logging
+import random
+import ipaddr
+import ipaddress
+import pytest
+
+import ptf.testutils as testutils
+import ptf.packet as scapy
+
+from ptf.mask import Mask
+from socket import INADDR_ANY
+
+pytestmark = [
+    pytest.mark.topology("t1")
+]
+
+logger = logging.getLogger(__name__)
+
+class DhcpPktFwdBase:
+    """
+    Base class for DHCP packet forwarding test. The test ensure that DHCP packets are going through T1 device.
+    """
```

Comment:
```suggestion
    """Base class for DHCP packet forwarding test. The test ensures that DHCP packets are going through T1 device."""
```

Since this is a one-liner we can compress it like so.